cloud/concepts.mdx

```diff
@@ -1,33 +1,56 @@
 ---
 title: Concepts
-description: This document describes the basic concepts of Flipt Enterprise Cloud.
+description: This document describes the basic concepts of Flipt Managed Cloud.
 ---
 
-Flipt Enterprise Cloud allows you to run Flipt on your infrastructure with the added benefits of a managed service. This document describes the basic concepts of Flipt Enterprise Cloud.
+<Info>
+  Flipt Managed Cloud is currently in beta. Some functionality may be subject to
+  change.
 
-## Enterprise Cloud
+Interested in joining the beta? [Request access here](https://flipt.cloud).
 
-With Flipt Enterprise Cloud, you deploy open-source Flipt on your infrastructure or cloud provider of choice as you always have, and we provide you with effortless ingress, enterprise-grade authentication and authorization. Experience a seamless transition with enhanced features that fit perfectly into your existing setup.
+</Info>
 
-## Gateways
+Flipt Managed Cloud is a fully managed solution where we host and operate Flipt for you. You no longer need to deploy or manage Flipt instances yourself. We provide a secure, scalable, and feature-rich environment with enterprise-grade authentication and authorization built-in.
```

Comment:
Is it worth re-clarifying that the option remains to self-host and they will work together seemlessly?